recipes/aaf/all/conanfile.py

```diff
@@ -41,50 +44,49 @@ def validate(self):
             raise ConanInvalidConfiguration("ARM v8 not supported")
```

Comment:
Apple Silicon is supported since 1.2.0 RC1: https://sourceforge.net/p/aaf/code2/ci/master/tree/